src/gluonts/dataset/parallelized_loader.py

```diff
@@ -84,16 +85,77 @@ def reduce_ndarray(data):
 ForkingPickler.register(nd.NDArray, reduce_ndarray)
 
 
+def _is_stackable(
+    arrays: List[Union[np.ndarray, mx.nd.NDArray, Any]],
```

Comment:
Is there any point in having this Union, if it contains Any?

Reply:
If I put `int`, `float` instead of `Any`, mypy complains despite the isinstance() check below. So, those are just meant to be "type hints" I guess.